the_mines/get_html.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+import requests
+import shutil
+
+
+def get_html():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--matchday", action="store", required=True, help="ex. 8",
+    )
+    parser.add_argument(
+        "--season", action="store", required=True, help="ex. 2019/2020",
+    )
+    args = parser.parse_args()
+
+    matchday = f"{args.matchday}"
+    season = f"{args.season}"
+
+    url = "https://www.fussballdaten.de/bundesliga/" + season[5:9] + "/" + matchday
```

Comment:
It may also be a good idea to make the target url a parameter (we can default it to `fussballdaten`).  That makes the method more robust against future urls we may want to hit.  A change like this would probably mean having some type of url constructor method to handle known sites.

Reply:
I agree. I think for now we should keep "https://www.fussballdaten.de/bundesliga/" hard-coded, but have an if statement to complete the rest of the URL depending on what stats we need.

Reply:
Ok, sounds good.